test/suite/test_rollback_to_stable10.py

```diff
@@ -76,6 +76,7 @@ def simulate_crash_restart(self, olddir, newdir):
         self.conn = self.setUpConnectionOpen(newdir)
         self.session = self.setUpSessionOpen(self.conn)
 
+    @unittest.skip("Skipping until the follow on work from WT-6280 is completed.")
```

Comment:
What is the failure mode here if you don't disable the test?

Reply:
This test fails as it expects to cleanup updates in rollback to stable which exist as a result of a race between eviction and checkpoint. That race can no longer happen as a result of this change. The test fails an assertion checking that it did clean up those updates which it didn't. This is all WT-6330 and WT-6375.

Reply:
I see - and it was reasonable for those updates to be there if there wasn't any use of mixed-mode timestamps, but with mixed-mode timestamps it's not safe because we add an update that inserts an immediately visible tombstone into the history store.
I'm starting to wonder whether we are solving a real or imaginary problem here. I've added a comment to the JIRA ticket exploring that further.